CHANGELOG.next.md

```diff
@@ -31,14 +28,25 @@ Thanks, you're awesome :-) -->
 
 #### Added
 
-* Add `parameters` property for field definitions, to provide any mapping parameter. #2084
-
 #### Improvements
 
-* Bump `gitpython` from `3.1.27` to `3.1.30` in `/scripts`. #2139
```

Comment:
This one not needed?

Reply:
It was listed in the [8.7 changelog](https://github.com/elastic/ecs/blob/main/CHANGELOG.md#improvements-1), so I checked the PR and it had the 8.7 label, so I removed it